base/libgit2/consts.jl

```diff
@@ -271,12 +274,14 @@ Option flags for `GitRepo`.
                         CREDTYPE_USERNAME           = Cuint(1 << 5),
                         CREDTYPE_SSH_MEMORY         = Cuint(1 << 6))
 
+if LibGit2.version() >= v"0.24.0"
```

Comment:
Yes, Rawhide has just switched to 0.24. I didn't have any new feature in mind other than following upstream (with the hope that the API will eventually stabilize). I don't see the point of dropping support for older versions, unless it we experience actual issues. As for whether 0.21 is still enough, I don't know...